src/OrchardCore/OrchardCore.Workflows.Abstractions/Display/ActivityDisplayDriver.cs

```diff
@@ -42,10 +42,8 @@ public override IDisplayResult Edit(TActivity model)
         public async override Task<IDisplayResult> UpdateAsync(TActivity model, IUpdateModel updater)
         {
             var viewModel = new TEditViewModel();
-            if (await updater.TryUpdateModelAsync(viewModel, Prefix))
-            {
-                await UpdateActivityAsync(viewModel, model);
-            }
+            await updater.TryUpdateModelAsync(viewModel, Prefix);
+            await UpdateActivityAsync(viewModel, model);
 
             return Edit(model);
```

Comment:
Here normally we should vall Edit(ViewModel) and this is why we need to return and update the model instead which is probably a design mistake.